### ISSUE TITLE
(Enhancement)Support extracting .zstd files from the desktop (gnome for example)

### ISSUE THREAD

**Issue by romulasry** [clearlinux/distribution#2154]
https://github.com/Frogging-Family/wine-tkg-git/releases/download/5.19.r1.gb49e8743/wine-tkg-staging-fsync-git-5.19.r1.gb49e8743-300-x86_64.pkg.tar.zst

**Comment by lebensterben** [comment]
check this: https://stackoverflow.com/questions/47235022/add-a-custom-option-in-nautilus-right-click-menu

This can be done as a user-customization and should not be made as a distro default behavior.

**Comment by romulasry** [comment]
No, it already has the option there, there just isn't support out of the box. There should be IMHO.

**Comment by lebensterben** [comment]
> it already has the option there
@romulasry What does that mean?

Common file types are defined by [shared-mime-info](https://freedesktop.org/wiki/Software/shared-mime-info/) package.

If you want `zstd` file to be supported "out-of-the box", you should instead make a PR to that package. Specifically, you should add the file type to `freedesktop.org.xml`.

**Comment by romulasry** [comment]
done
https://gitlab.freedesktop.org/xdg/shared-mime-info/-/issues/145

**Comment by lebensterben** [comment]
This is hardly "done". Can you just make an PR instead of asking others to spend time on fulfilling your needs?
